automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/ClusterConcurrencySpec.scala

```diff
@@ -138,6 +138,43 @@ class ClusterConcurrencySpec extends FreeSpec with LeonardoTestUtils with Parall
       }
 
     }
+
+    // make sure adding a worker works
```

Comment:
Good test but I'm not sure it belongs in `ClusterConcurrencySpec`. Maybe we could put it in `ClusterMonitoringSpec`.